example_test.go

```diff
@@ -52,3 +53,25 @@ func Example() {
 	// http_request_seconds_created{status="200"} 1515151515.757576
 	// # EOF
 }
+
+func ExampleExemplar() {
+	reg := openmetrics.NewConsistentRegistry(mockNow) // or, openmetrics.DefaultRegistry()
+	internalError := reg.Counter(openmetrics.Desc{
+		Name: "internal_error",
+		Help: "A counter example",
+	})
+
+	_, err := os.Stat("/.dockerenv")
+	if err != nil {
+		// the combined length of label names and values of MUST NOT exceed 128 characters
+		errmsg := err.Error()
+		if len(errmsg) > 120 {
+			errmsg = errmsg[:120]
+		}
```

Comment:
IMO, this can become quite repetitive.

Can this be done right in Exemplar.AddLabel or even in LabelSet.Append? (filter/shorten them)

Ideally with some logging `label "foo" truncated, was: "FULL MSG"`

It's maybe complication for the lib, but IMO it's good for lib user.

Reply:
sorry, I don't want to overcomplicate things. exemplars are used rarely and I prefer not to add more than is necessary